types/nationalAvalancheCenter/schemas.ts

```diff
@@ -25,6 +25,28 @@ export function userFacingCenterId(input: AvalancheCenterID): string {
   return input === 'SNFAC' ? 'SAC' : input;
 }
 
+export const AvalancheCenterWebsites: Record<AvalancheCenterID, string> = {
+  ['NWAC']: 'https://nwac.us/',
+  // ['BAC']: 'https://bridgeportavalanchecenter.org/',
```

Comment:
what would be the best way here? To only populate centers that we support for sharing or implement some sort of other way (a boolean that quantifies if sharing status is on or off?) 

I thought of just having values be uncommented as we add them, but right now the enum wants all AvalancheCenterID values in here. And it made me wonder if that was actually the best way 🤔

Reply:
thats a good point - we can still share links from the app, they just wont open in the app for the centers that do not yet have support for opening links in the app.